services/clusters-keeper/src/simcore_service_clusters_keeper/modules/clusters_management_core.py

```diff
@@ -149,7 +169,72 @@ async def check_clusters(app: FastAPI) -> None:
         )
         await delete_clusters(app, instances=terminateable_instances)
 
-    # the other instances are broken (they were at some point connected but now not anymore)
+    # TODO: transmit command to start docker swarm/stack if needed
```

Comment:
TODO or not TODO? ;-)

Reply:
yep, good catch I need to remove it. I'm worried, I did not see an entry in sonar/?